tests/components/derivative/test_sensor.py

```diff
@@ -97,132 +70,88 @@
     state = hass.states.get("sensor.power")
     assert state is not None
 
-    assert round(float(state.state), config["sensor"]["round"]) == -0.5
+    assert round(float(state.state), config["sensor"]["round"]) == expected_state
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4938379)